### PR TITLE
[SDPA-2911] Update Readme

### PR DESCRIPTION
**Pull request by dylankelly**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-

### Changed

1.  Updated Readme formatting.

### CODE REVIEW

**Comment by tim-yao** [comment]
@dylankelly any place you think we can add this https://github.com/dpc-sdp/ripple/issues/77 ?

File: README.md [hunk 1]
```diff
@@ -1,26 +1,119 @@
...
+
+- Install [nodejs](https://nodejs.org/en/) (^v10.0.0)
+- Install [yarn](https://yarnpkg.com/en/docs/install) (currently npm is unsupported)
+
+### Install
 
 Run `yarn install`
 
-## Running Example
+### Running Example Application
```
Comment (tim-yao) [inline README.md]: I think this part is belong to 'CONTRIBUTING', as this is for local env setup of core devs work.

Users of Ripple should just want to know how to install Ripple into their project.

File: README.md [hunk 2]
```diff
@@ -1,26 +1,119 @@
...
+
+The example application requires a working [Tide](https://github.com/dpc-sdp/tide) Drupal installation to connect to.
 
 Ensure a populated `.env` exists within the `/examples/vic-gov-au/` directory.
 You can use `/examples/vic-gov-au/example.env` as a template.
 
 Run `yarn run start:example`
 
-## Running Storybook
+### Running Storybook
```
Comment (tim-yao) [inline README.md]: Same as above, I think this is "CONTRIBUTING" doc.

File: README.md (hunk 1 above)
Comment (dylankelly) [reply README.md]: They will not interact with "Ripple" they will interact with "@dpc-sdp/ripple-nuxt-tide" Which is where that documentation should be. But I have updated the Ripple README.md with basic usage for each package. I think this should be an overview of everything with links for more detail as required.

File: README.md (hunk 2 above)
Comment (dylankelly) [reply README.md]: See my comment above

**Pull request merged**